scripts-dev/release.py

```diff
@@ -296,7 +301,9 @@ def tag(gh_token: Optional[str]):
 
     click.echo_via_pager(changes)
     if click.confirm("Edit text?", default=False):
-        changes = click.edit(changes, require_save=False)
+        edited_changes = click.edit(changes, require_save=False)
+        assert edited_changes is not None
+        changes = edited_changes
```

Comment:
The `click` docs suggest that `edited_changes` would be `None` when the user closes the editor without saving: https://click.palletsprojects.com/en/7.x/utils/#launching-editors

Do we want to abort via an assert or use the unedited changes in that case?

Reply:
I found the docs hard to read. It seems like there might be 

Testing with `print(click.edit("Hello", require_save=...))` yielded:

|                      | require_save: False | requre_save: True |
|----------------------|---------------------|-------------------|
| **quit without save**    | original text       | None              |
| **save without changes** | original text       | original text     |
| **save with  changes**   | changed text        | changed text      |

So I think this assert is for mypy's benefit only. Perhaps it would be clearer to cast this to `str` with a comment?

Reply:
Curious. In that case I'm happy with the assert!